frontend/src/js/ui-components/InputDate.tsx

```diff
@@ -0,0 +1,114 @@
+import styled from "@emotion/styled";
+import { createElement, createRef, forwardRef, useState } from "react";
+import ReactDatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+import { formatDate, parseDate } from "../common/helpers/dateHelper";
+
+import BaseInput, { Props as BaseInputProps } from "./BaseInput";
+
+const Root = styled("div")`
+  position: relative;
+
+  .react-datepicker-wrapper {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    z-index: -1;
+  }
+  .react-datepicker-popper[data-placement^="bottom"] {
+    padding-top: 0.25rem;
+    transform: translate3d(0, 2rem, 0) !important;
+  }
+  .react-datepicker-popper[data-placement^="top"] {
+    padding-bottom: 0;
+    transform: translate3d(0, -2rem, 0) !important;
+  }
+`;
+
+const HiddenInput = styled("input")`
+  display: none;
+`;
+
+const StyledCalendar = styled("div")`
+  .react-datepicker__day--selected {
+    background: ${({ theme }) => theme.col.green};
```

Comment:
Due to legacy our primary color is called `blueGrayDark`, haha! How about using it here:
```suggestion
    background: ${({ theme }) => theme.col.blueGrayDark};
    font-weight: 700;
```